schema.sql

```diff
@@ -3,9 +3,10 @@
 CREATE DATABASE vet_clinic;
 CREATE TABLE animals (
     id BIGSERIAL NOT NULL,
-    name VARCHAR(50),
-    date_of_birth DATE,
-    escape_attempts INT,
-    neutered BOOLEAN,
-    weight_kg DECIMAL
+    name VARCHAR(50) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT NOT NULL,
+    neutered BOOLEAN NOT NULL,
+    weight_kg DECIMAL NOT NULL,
+    species VARCHAR(120)
```

Comment:
- [ ] @Nejjari001 while you have made great effort adding the species column, you used the wrong method, kindly remove it and add it using the ALTER keyword, this is because when adding the data in the database we are not supposed to manually alter the schema but use ALTER keyword to add edit or remove a column and its properties.